assets/js/modules/analytics/setup.js

```diff
@@ -87,11 +85,39 @@ class AnalyticsSetup extends Component {
 	async componentDidMount() {
 		this._isMounted = true;
 
-		this.iframeLoad();
+		await this.getAccounts().then( async( newState ) => {
 
-		await this.getAccounts();
+			// If tag hasn't been detected in wp_head, look for existing tag in the html.
```

Comment:
I believe we don't detect tag in wp_head anymore

Reply:
@ivankristianto we are still detecting in this PR, opened another issue to follow this PR on removing wp_head detection https://github.com/google/site-kit-wp/issues/83